libs/labelbox/src/labelbox/schema/ontology.py

```diff
@@ -221,6 +221,183 @@ def add_option(self, option: Option) -> None:
                 f"Duplicate option '{option.value}' "
                 f"for classification '{self.name}'.")
         self.options.append(option)
+        
+@dataclass
+class ResponseOption:
```

Comment:
this looks like `Option` but renamed. Any particular reasons for duplication? Couldn't `ResponseOption` be just alias for `Option`?

Reply:
I added it for consistency to separate it from normal classifications. Prompt response classifications are different from normal classifications (names and some params), and I want users to understand that, but I switched it to an alias since you are right. It has a lot of duplication.